internal/service/alertconfiguration/resource_alert_configuration_test.go

```diff
@@ -421,6 +422,32 @@ func TestAccConfigRSAlertConfiguration_withPagerDuty(t *testing.T) {
 	})
 }
 
+func TestAccConfigAlertConfiguration_PagerDutyUsingIntegrationID(t *testing.T) {
+	acc.SkipTestForCI(t) // currently skipped as third party integration id cannot be created and obtained during test
+	proxyPort := replay.SetupReplayProxy(t)
+
+	var (
+		pagerDutyIntegrationID = os.Getenv("MONGODB_ATLAS_PAGER_DUTY_THIRD_PARTY_INTEGRATION_ID")
+		projectID              = replay.ManageProjectID(t, acc.ProjectIDExecution)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckPagerDutyIntegrationID(t); acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6FactoriesWithProxy(proxyPort),
+		CheckDestroy:             checkDestroyUsingProxy(proxyPort),
+		Steps: []resource.TestStep{
+			{
+				Config: configWithPagerDutyIntegrationID(projectID, pagerDutyIntegrationID),
+				Check: resource.ComposeTestCheckFunc(
+					checkExistsUsingProxy(proxyPort, resourceName),
+					resource.TestCheckResourceAttr(resourceName, "notification.0.integration_id", pagerDutyIntegrationID),
```

Comment:
when you enable the test, be aware that plural ds shouldn't assume any result order as it's happening here (if using  ProjectIDExecution)